skops/hub_utils/_hf_hub.py

```diff
@@ -427,6 +430,14 @@ def init(
             model_format=model_format,
             use_intelex=use_intelex,
         )
+
+        if model_format == "pickle":
+            with open(model, "rb") as f:
+                model = pikle_load(f)
```

Comment:
please write a test to cover these lines.

Reply:
Initally, the test was written for those lines (without the if-statement) in function `test_override_init_modelcard`. Now we are accounting for the possibility of the `skops` format. However, I believe that both formats are already tested since the pytest fixture for `config_json` in `test_hf_hub.py` provides both formats in its `params` argument. Please correct me if I'm wrong.